docs/using-flatpak.rst

```diff
@@ -62,17 +78,26 @@ The same principle applies to repositories - repositories that have been
 added system-wide are available to all users, whereas per-user repositories
 can only be used by a particular user.
 
-Flatpak commands are run system-wide by default. If you are installing
-applications for day-to-day usage, it is recommended to stick with this
-default behavior.
+.. Flatpak commands are run system-wide by default. If you are installing
+.. applications for day-to-day usage, it is recommended to stick with this
+.. default behavior.
+
+Flatpak commands are run system-wide by default. If you are installing 
+applications for day-to-day usage, it is recommended to use ``--user`` if 
+you don't need your apps available for all users on your system.
 
-However, running commands per-user can be useful for testing and development
+Running commands per-user can be useful for testing and development
 purposes, since objects that are installed in this way won't be available
 to other users on the system. To do this, use the ``--user`` option, which
 can be used in combination with most ``flatpak`` commands.
 
 Commands behave in exactly the same way if they are run per-user rather
-than system-wide.
+than system-wide, but running them system-wide may require root permissions.
```

Comment:
It does not require root permissions. Polkit is used to authenticate the user which is a bit more complicated.